### DISCUSSION TITLE
Can I access all available options of an avatar style via REST API?

### DISCUSSION THREAD

**Discussion by vanthomiy** [dicebear/dicebear#337] [Q&A] [answered]
I need to access available options for a style via REST API.

An example how to access is provided for a TS/JS here: [https://www.dicebear.com/guides/access-all-available-options](https://www.dicebear.com/guides/access-all-available-options)

Actually I dont find any information how to achive this via REST API.

Can someone help me?

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @vanthomiy 

Added an endpoint for the API to output schema.json. Experimental feature.

Example: https://api.dicebear.com/6.x/adventurer/schema.json
